lldb/include/lldb/Interpreter/Interfaces/ScriptedInterface.h

```diff
@@ -68,6 +68,13 @@ class ScriptedInterface {
     return true;
   }
 
+  static bool
+  CreateInstance(lldb::ScriptLanguage language,
+                 std::vector<llvm::StringRef> command_interpreter_usages,
+                 std::vector<llvm::StringRef> api_usages) {
+    return false;
+  }
```

Comment:
Without `GetScriptedInterfaceCreateCallbackAtIndex` I don't think you need this either.